src/main/java/com/unstoppabledomains/util/DnsRecord.java

```diff
@@ -0,0 +1,21 @@
+package com.unstoppabledomains.util;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.ToString;
+
+@Getter
+@AllArgsConstructor
+@ToString
+public class DnsRecord {
+  private DnsRecordsType type;
+  private Number ttl;
+  private String data;
+
+  @Override
+  public boolean equals(Object obj) {
+    DnsRecord that = (DnsRecord) obj;
+
+    return that.type.equals(type) && that.ttl.equals(ttl) && that.data.equals(data);
+  }
```

Comment:
You must also override `hashCode()` if you're doing this for `equals()`. 
But here you can just do an `@EqualsAndHashCode` on a class-level instead of this block.